introduction.md

```diff
@@ -1 +1,5 @@
-My favorite Code School path is Electives
+* My favorite Code School path is Electives
+* Git
+* Bootstrap
+* R
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?